Sources/PackageDescription/PackageDescription.swift

```diff
@@ -295,11 +347,10 @@ public final class Package {
     ///   - traits: The set of traits of this package.
     ///   - dependencies: The list of package dependencies.
     ///   - targets: The list of targets that are part of this package.
-    ///   - swiftLanguageVersions: The list of Swift versions with which this package is compatible.
+    ///   - swiftLanguageModes: The list of Swift language modes with which this package is compatible.
     ///   - cLanguageStandard: The C language standard to use for all C targets in this package.
     ///   - cxxLanguageStandard: The C++ language standard to use for all C++ targets in this package.
-    @_spi(ExperimentalTraits)
```

Comment:
`@_spi` needs to stay on this one with same availability annotation as before, since traits are experimental and not available for any valid tools version.

Reply:
Done!